### PR TITLE
making low_speed_limit and low_speed_time configurable

### PR DESCRIPTION
**Pull request by doccaz**
Implementing the changes described on issue #493 .

### CODE REVIEW

**Comment by ikapelyukhin** [comment]
Making those configurable makes sense to me. I also think it would make sense to also add those options to the default config (`/package/obs/rmt.conf`), just so that it is obvious that those options exist.

(We'd also need to document those in RMT handbook, but that's an action item for when the PR is merged)

File: lib/rmt/http_request.rb [hunk 1]
```diff
@@ -17,8 +17,8 @@ def set_defaults
     end
 
     # Abort download if speed is below 512 bytes/sec for 120 sec, to prevent downloads from getting stuck
-    options[:low_speed_limit] = 512
-    options[:low_speed_time] = 120
+    options[:low_speed_limit] = Settings.try(:http_client).try(:low_speed_limit) ? Settings.try(:http_client).try(:low_speed_limit) : 512
```
Comment (thutterer) [inline lib/rmt/http_request.rb]:
```suggestion
    options[:low_speed_limit] = Settings.try(:http_client).try(:low_speed_limit) || 512
```

File: lib/rmt/http_request.rb [hunk 2]
```diff
@@ -17,8 +17,8 @@ def set_defaults
...
+    options[:low_speed_time] = Settings.try(:http_client).try(:low_speed_time) ? Settings.try(:http_client).try(:low_speed_time) : 120
```
Comment (thutterer) [inline lib/rmt/http_request.rb]:
```suggestion
    options[:low_speed_time] = Settings.try(:http_client).try(:low_speed_time) || 120
```

File: lib/rmt/http_request.rb [hunk 3]
```diff
@@ -17,8 +17,8 @@ def set_defaults
     end
 
     # Abort download if speed is below 512 bytes/sec for 120 sec, to prevent downloads from getting stuck
```
Comment (thutterer) [inline lib/rmt/http_request.rb]: Maybe update this comment since the values aren't hard-coded anymore.

**Comment by doccaz** [comment]
Sorry for the delay.  Here are all the changes you requested.

**Comment by felixsch** [comment]
Hi doccaz,

Thank you for you contribution! The changes look good to me. The only thing would it be possible for you to update the tests in `spec/lib/rmt/http_request_spec.rb`. They assume a hardcoded value which is now configurable.

In case there are questions how to run the tests or anything else let me know.

**Comment by felixsch** [comment]
Hi doccaz,

to not longer delay the pull request. I decided to merge your and write the test myself. Thank you for your contribution again!

**Comment by doccaz** [comment]
I'm so sorry! I delved into a critical problem at a customer for the whole
week and completely forgot about this. Thank you so much!

**Pull request merged**